src/core_plugins/kibana/public/discover/components/field_chooser/discover_field.js

```diff
@@ -95,6 +93,13 @@ app.directive('discoverField', function ($compile) {
         }
       };
 
+      $scope.onKeyUp = function onKeyUp(e, field) {
+        // Support keyboard accessibility by toggling display on ENTER or SPACE keypress.
+        if ([13, 32].includes(e.keyCode)) {
+          $scope.toggleDetails(field);
+        }
+      };
```

Comment:
Thanks for bringing this up! I think it's a great idea but I'm a little stumped for the best place and name for this module. Suggestions?